tdvt/tdvt/config/tdvt/tdvt.ini

```diff
@@ -1,3 +1,3 @@
 [DEFAULT]
-TAB_CLI_EXE_X64 = C:/Program Files/Tableau/Tableau 2019.2/bin/tabquerytool.exe
-TAB_CLI_EXE_MAC = /Applications/Tableau 2019.2.app/Contents/MacOS/tabquerytool
\ No newline at end of file
+TAB_CLI_EXE_X64 = C:/Program Files/Tableau/Tableau 2019.3/bin/tabquerytool.exe
```

Comment:
should we change all of these references to 2019.4?

Reply:
Maybe we could set them to something non-specific and then rely on setting the env variable when running the tests locally.

Reply:
Good idea, I reverted the changes.